src/contrib/cluster/Akka.Cluster.Sharding/ShardingMessages.cs

```diff
@@ -153,6 +158,77 @@ private GetCurrentRegions()
         }
     }
 
+    /// <summary>
+    /// Send this message to a <see cref="ShardRegion"/> actor to determine the location and liveness
+    /// of a specific entity actor in the region.
+    ///
+    /// Creates a <see cref="EntityLocation"/> message in response.
+    /// </summary>
+    /// <remarks>
+    /// This is used primarily for testing and telemetry purposes.
+    ///
+    /// In order for this query to work, the <see cref="MessageExtractor"/> must support <see cref="ShardRegion.StartEntity"/>,
+    /// which is also used when remember-entities=on.
+    /// </remarks>
+    public sealed class GetEntityLocation : IShardRegionQuery
```

Comment:
This message never gets sent over the wire, so it doesn't require any changes to the wire type nor does it need to support `IClusterShardingSerializable`.